pom.xml

```diff
@@ -2400,7 +2400,6 @@
       </build>
     </profile>
 
-    <!-- for those using JDK 11 -->
```

Comment:
Does the activation need to change here? presently ``(1.8,)``

Reply:
Updated